apps-rendering/src/components/Layout/MediaLayout.tsx

```diff
@@ -59,6 +59,7 @@ const MediaLayout: FC<Props> = ({ item, children }) => {
 							publicationDate={item.publishDate}
 							className={articleWidthStyles}
 							item={item}
+							edition={item.edition}
```

Comment:
If this already takes `item` does it also need to take `edition`?

Reply:
It doesn't! I'll remove the `edition` prop